pkg/pipeline/watch.go

```diff
@@ -121,6 +121,7 @@ func (c *Controller) messageWatch(msg *gst.Message) bool {
 	switch msg.Type() {
 	case gst.MessageEOS:
 		logger.Infow("EOS received")
+		c.eosTimer.Stop()
```

Comment:
nil check? possible EOS is sent by something else?

Reply:
shouldn't be possible but might as well add it